src/KuzzleError.ts

```diff
@@ -81,6 +126,9 @@ export class KuzzleError extends Error {
       this.stack = apiError.stack + '\n';
       this.stack += '          |\n';
       this.stack += '          |\n';
+
+      this.stack += !this.controller && !this.action ? '' : `          Http: ${this.controller}:${this.action}\n`;
+
       this.stack += `          ${protocol}\n`;
```

Comment:
This line is redundant with the previous one, we don't need to specify the protocol twice

Reply:
done